src/components/mandataires/indiPrepo.js

```diff
@@ -0,0 +1,116 @@
+import dynamic from "next/dynamic";
+import { Home, Map, UserMinus, Clock, FilePlus } from "react-feather";
+
+/* TEMP : the redux store will be moved at root level */
```

Comment:
Hum... 
How did write this comment ? you ?

PS: It's strange that it's multiline BTW...